plugins/modules/k8s_service.py

```diff
@@ -195,51 +187,12 @@ def argspec():
     return argument_spec
 
 
-def execute_module(module, k8s_ansible_mixin):
-    """ Module execution """
-    k8s_ansible_mixin.set_resource_definitions(module)
-
-    api_version = "v1"
-    selector = module.params.get("selector")
-    service_type = module.params.get("type")
-    ports = module.params.get("ports")
-
-    definition = defaultdict(defaultdict)
-
-    definition["kind"] = "Service"
-    definition["apiVersion"] = api_version
-
-    def_spec = definition["spec"]
```

Comment:
@gravesm `def_spec` and `def_meta` seem never to be used anywhere.

Reply:
They are used, but in a way that's not obvious at first glance. Since those reference dictionaries, they are assigned by reference. Changes in those dictionaries are reflected in the `definition` which is used later.

Reply:
Ah, ok! Thanks.